src/main/java/org/opentripplanner/routing/trippattern/TripTimes.java

```diff
@@ -28,6 +24,8 @@
     private static final long serialVersionUID = 1L;
     private static final Logger LOG = LoggerFactory.getLogger(TripTimes.class);
 
+    private FeedScopedId id;
```

Comment:
I think that this class need to extend TransitEntity, but we can add it when needed. So we should probably have some startegy for generating ids for none DSJ entries.  It is needed to be able to attach Noticies and Alerts.